src/network/connection.cpp

```diff
@@ -505,7 +505,7 @@ void IncomingSplitBuffer::removeUnreliableTimedOuts(float dtime, float timeout)
 				remove_queue.push_back(i.first);
 		}
 	}
-	for (u16 j : remove_queue) {
+	for (const auto &j : remove_queue) {
```

Comment:
Taking a reference of a tiny variables will be less efficient than a copy

Reply:
@sfan5, yes, that's right, type is too small, inattentively changed to `const &`

Reply:
fix it https://github.com/minetest/minetest/pull/12704/commits/50efdb084bf3c4a1af5757106c26a1ccbf34134d